index.html

```diff
@@ -63,33 +64,33 @@
 		<div class="Section Section--bordered">
 			<div class="Section__full">
 				<div class="Carousel">
-					<div class="Carousel__arrow-left"> < </div>
+					<!-- Having it be < seemed to screw up the structure -->
+					<div class="Carousel__arrow-left"> Left </div>
 					<div class="Carousel__items">
 						<div class="Carousel__item Carousel__item-focused">
 							<div class="Box Box--placeholder Box--grey">
 								<div class="Box__title">Box # 1</div>
 								<div class="Box__description">Lorem ipsum dolor sit amet, consectetur adipiscing elit. Fusce risus nibh, gravida nec felis quis, facilisis facilisis lectus. Nulla ac orci pretium, condimentum orci quis, accumsan nisi. Aliquam erat volutpat. Curabitur cursus mattis libero, at viverra risus hendrerit quis. Fusce imperdiet tristique tortor non tincidunt. Mauris accumsan urna nec augue feugiat porta. Proin vitae magna in ex malesuada laoreet eget a nulla. Aliquam tristique et elit at consequat. In hac habitasse platea dictumst.</div>
 							</div>
 						</div>
-					</div>
 					<div class="Carousel__item">
 							<div class="Box Box--placeholder Box--blue">
-								<div class="Box__title">Box # 1</div>
+								<div class="Box__title">Box # 2</div>
 								<div class="Box__description">Lorem ipsum dolor sit amet, consectetur adipiscing elit. Fusce risus nibh, gravida nec felis quis, facilisis facilisis lectus. Nulla ac orci pretium, condimentum orci quis, accumsan nisi. Aliquam erat volutpat. Curabitur cursus mattis libero, at viverra risus hendrerit quis. Fusce imperdiet tristique tortor non tincidunt. Mauris accumsan urna nec augue feugiat porta. Proin vitae magna in ex malesuada laoreet eget a nulla. Aliquam tristique et elit at consequat. In hac habitasse platea dictumst.</div>
 							</div>
 						</div>
-					</div>
 					<div class="Carousel__item">
 							<div class="Box Box--placeholder Box--green">
-								<div class="Box__title">Box # 1</div>
+								<div class="Box__title">Box # 3</div>
 								<div class="Box__description">Lorem ipsum dolor sit amet, consectetur adipiscing elit. Fusce risus nibh, gravida nec felis quis, facilisis facilisis lectus. Nulla ac orci pretium, condimentum orci quis, accumsan nisi. Aliquam erat volutpat. Curabitur cursus mattis libero, at viverra risus hendrerit quis. Fusce imperdiet tristique tortor non tincidunt. Mauris accumsan urna nec augue feugiat porta. Proin vitae magna in ex malesuada laoreet eget a nulla. Aliquam tristique et elit at consequat. In hac habitasse platea dictumst.</div>
 							</div>
 						</div>
 					</div>
-					<div class="Carousel__arrow-right"> > </div>
+					<div class="Carousel__arrow-right"> > </div>	
```

Comment:
You could spice things a little bit and add few CSS rules to improve Carousel UI for left and right arrow. `cursor: pointer` and `user-select: none` would make things look a lot better.